tests/h/links_test.py

```diff
@@ -23,6 +23,32 @@ def __init__(self):
         self.document_uris = []
 
 
+@pytest.mark.usefixtures('routes')
+class TestHTMLLink(object):
+
+    def test_html_link_returns_links_for_first_party_annotations(
+            self, factories, pyramid_request):
+        annotation = factories.Annotation(
+            # Specify the username in order to make sure that it's a first-party
+            # annotation.
+            userid='acct:fred@example.com',
```

Comment:
TODO: I'd rather just pass `authority` here rather than the entire `userid` because the authority part is all that I care about. Or even call `factories.FirstPartyAnnotation()` or `factories.ThirdPartyAnnotation()`. But this will require adding to the annotation factories